.gitignore

```diff
@@ -6,3 +6,6 @@ vscode/tsbuildinfo
 vscode/.metadata
 vscode/org.eclipse*
 .aider*
+node_modules
+webview-ui/node_modules
+webview-ui/build
```

Comment:
Should be enough to do just:
```
node_modules/
build/
```
and have directories with those names ignored by git from this folder down